packages/extension/src/companion/CompanionContextMenu.tsx

```diff
@@ -79,6 +83,21 @@ export default function CompanionContextMenu({
         className="menu-primary"
         animation="fade"
       >
+        <Item
+          onClick={onDownvote}
+          aria-pressed={!!postData?.downvoted}
+          aria-label={postData?.downvoted ? 'Remove downvote' : 'Downvote'}
+        >
+          <DownvoteIcon
+            size={IconSize.Small}
+            className={classNames(
+              'mr-2',
+              postData?.downvoted && 'text-ketchup-40',
```

Comment:
I think we should the primary-ketchup.
```suggestion
              postData?.downvoted && 'text-theme-color-ketchup',
```